doc/build-instructions/Build_Instructions_V15.md

```diff
@@ -0,0 +1,627 @@
+
+1. The first thing you need to do is install Docker. You can download the free Community edition from [here](https://docs.docker.com/engine/installation/), which also contains instructions for installing Docker on your system.  You should also read the [Getting started](https://docs.docker.com/get-started/) guide to familiarise yourself with the basic Docker concepts and terminology.
+
+2. Obtain the [docker build script](https://github.com/eclipse/openj9/blob/master/buildenv/docker/mkdocker.sh) to build and run a container that has all the correct software pre-requisites.
+
+Download the docker build script to your local system or copy and paste the following command:
+
+```
+wget https://raw.githubusercontent.com/eclipse/openj9/master/buildenv/docker/mkdocker.sh
+```
+
+3. Next, run the following command to build a Docker image, called **openj9**:
+```
+bash mkdocker.sh --tag=openj9 --dist=ubuntu --version=16.04 --build
+```
+
+4. Start a Docker container from the **openj9** image with the following command, where `-v` maps any directory, `<host_directory>`,
+on your local system to the containers `/root/hostdir` directory so that you can store the binaries, once they are built:
+```
+docker run -v <host_directory>:/root/hostdir -it openj9
+```
+
+:pencil: Depending on your [Docker system configuration](https://docs.docker.com/engine/reference/commandline/cli/#description), you might need to prefix the `docker` commands with `sudo`.
+
+Now that you have the Docker image running, you are ready to move to the next step, [Get the source](#2-get-the-source).
+
+#### Setting up your build environment without Docker
+
+If you don't want to user Docker, you can still build directly on your Ubuntu system or in a Ubuntu virtual machine. Use the output of the following command like a recipe card to determine the software dependencies that must be installed on the system, plus a few configuration steps.
+
+```
+bash mkdocker.sh --tag=openj9 --dist=ubuntu --version=16.04 --print
+```
+
+1. Install the list of dependencies that can be obtained with the `apt-get` command from the following section of the Dockerfile:
+```
+apt-get update \
+  && apt-get install -qq -y --no-install-recommends \
+    ...
+```
+
+2. The previous step installed g++-7 and gcc-7 packages, which might be different
+than the default version installed on your system. Export variables to set the
+version used in the build.
+```
+export CC=gcc-7 CXX=g++-7
+```
+
+3. Download and setup **freemarker.jar** into a directory.
+```
+cd /<my_home_dir>
+wget https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download -O freemarker.tgz
+tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2
+rm -f freemarker.tgz
+```
+
+4. Download and setup the boot JDK using the latest AdoptOpenJDK v15 build.
+```
+cd /<my_home_dir>
+wget -O bootjdk15.tar.gz "https://api.adoptopenjdk.net/v3/binary/latest/15/ga/linux/x64/jdk/openj9/normal/adoptopenjdk"
```

Comment:
This isn't going to work until after Java 15 released, but using 14 is an option. This comment applies to all platforms.

Reply:
i.e. --with-boot-jdk=<path_to_boot_JDK15>

Reply:
We used to always pre-req the earlier release for the boot JDK, not sure when this changed. Have modified all instances.